extensions/eclipse-che-theia-plugin-ext/src/plugin/che-workspace.ts

```diff
@@ -0,0 +1,137 @@
+/*********************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { RPCProtocol } from '@theia/plugin-ext/lib/api/rpc-protocol';
+import { PLUGIN_RPC_CONTEXT, CheApiMain, FactoryAction as CheFactoryAction, Factory as CheFactory, ProjectConfig } from '../common/che-protocol';
+import { WorkspaceSettings, WorkspaceConfig, Workspace, ResourceCreateQueryParams, Factory, Project, FactoryAction, FactoryActionProperties } from '@eclipse-che/plugin';
```

Comment:
might be easier to readability to use
```
 import * as che from '@eclipse-che/plugin' (or something else than 'che')
```
and use che.Factory in the code

Reply:
OK